src/CoreBundle/Bridge/Chameleon/Module/NavigationTreeSingleSelect/NavigationTreeSingleSelect.php

```diff
@@ -33,14 +45,41 @@ class NavigationTreeSingleSelect extends MTPkgViewRendererAbstractModuleMapper
      * @var InputFilterUtilInterface
      */
     private $inputFilterUtil;
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @var UrlUtil
+     */
+    private $urlUtil;
     /**
      * @var BackendTreeNodeFactory
      */
     private $backendTreeNodeFactory;
+
+    /**
+     * @var string
+     */
+    private $fieldName;
```

Comment:
Does it need these new instance variables?

(So set in one method and then used later in a totally different one?)

Reply:
In fact, it was only used globally for createBreadcrumbStorage. Here I can also pass the fieldName as a parameter.

-> new commit